src/herder/HerderImpl.cpp

```diff
@@ -2132,14 +2120,6 @@ HerderImpl::start()
     }
 
     restoreUpgrades();
-    // make sure that the transaction queue is setup against
-    // the lcl that we have right now
-    mTransactionQueue.maybeVersionUpgraded();
```

Comment:
Why was this removed? I assume the current ledger version is no longer required after all the refactoring we've been doing?

Reply:
This is dead code at this point, as it's dealing with v13 protocol upgrade (this will never happen on a production network)